db.go

```diff
@@ -13,14 +13,16 @@ func OpenDBWithDriver(driver string, dbstring string) (*sql.DB, error) {
 	}
 
 	switch driver {
+	case "mssql":
+		driver = "sqlserver"
 	case "redshift":
 		driver = "postgres"
 	case "tidb":
 		driver = "mysql"
 	}
 
 	switch driver {
-	case "postgres", "sqlite3", "mysql", "mssql":
+	case "postgres", "sqlite3", "mysql", "sqlserver":
```

Comment:
For reference:
https://github.com/denisenkom/go-mssqldb#deprecated
https://github.com/denisenkom/go-mssqldb/blob/master/mssql.go#L30